modules/Core/includes/endpoints/CreateWebhooksEndpoint.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * @param string $name The name of the webhook
+ * @param string $url The url of the webhook
+ * @param string $type The webhook type
+ * @param array $events A list of events the webhook should receive
+ *
+ * @return string JSON Array
+ */
+class CreateWebhooksEndpoint extends KeyAuthEndpoint {
+
+    public function __construct() {
+        $this->_route = 'webhooks/create';
+        $this->_module = 'Core';
+        $this->_description = 'Create a new webhook';
+        $this->_method = 'POST';
+    }
+
+    public function execute(Nameless2API  $api): void {
+        // Validation
+        $validation = Validate::check($_POST, [
```

Comment:
Validation missing "events" perhaps also make Validation check IS ARRAY?

Atleast these errors are currently not used with they should be
public const ERROR_WEBHOOK_INVALID_TYPE = 'core:webhook_type_invalid';
public const ERROR_WEBHOOK_INVALID_EVENT = 'core:webhook_event_invalid';

Reply:
Added validation for these terms